spec/main_spec.rb

```diff
@@ -4,14 +4,18 @@
 
 require_relative '../lib/main'
 
-describe Fibonacci do 
-    subject(:fibonacci) {described_class.new}
+describe Puzzle do 
+    subject(:puzzle) {described_class.new}
 
     it ' has a class called fibonacci' do
-        expect(fibonacci).to be_instance_of(Fibonacci)
+        expect(puzzle).to be_instance_of(Puzzle)
     end
 
-    it 'takes an array as an argument and returns third-to-last and second-to-last numbers'do 
-       new_arr
+    it 'it should take an integer and return the fibonacci sequence up to that integer'do 
```

Comment:
I'm new to the fibonacci kata, and this test helped me to understand what it is, so thanks! :D

Reply:
Thank you 😄